aptos-move/block-executor/src/view.rs

```diff
@@ -682,6 +686,7 @@ impl<'a, T: Transaction, X: Executable> ResourceGroupState<T> for ParallelState<
                     // If we have a known layout, upgrade RawFromStorage value to Exchanged.
                     match value_with_layout {
                         ValueWithLayout::RawFromStorage(v) => {
+                            // TODO: we should halt if patch_base_value fails!
```

Comment:
same, let's also mark the TODO w. agg_v2

Reply:
doesn't patch_base_value()? propagate how patch_base_value defines errors? so we just need patch_base_value to return correct errors?

or you had something else in mind here?

Reply:
Missed that we can make patching return `DELAYED_FIELDS_CODE_INVARIANT_ERROR`. Not a todo anymore.